src/plonemeeting/portal/core/browser/sync.py

```
@@ -454,23 +487,36 @@ def _fetch_preview_items(context, meeting_external_uid):  # pragma: no cover
     response = _call_delib_rest_api(url, context)
     return json.loads(response.text)
 
 
 def _sync_meeting(
-    institution, meeting_uid, request, force=False, with_annexes=True, item_external_uids=[]
+    institution,
+    meeting_uid,
+    request,
+    force=False,
+    with_annexes=True,
+    item_external_uids=[],
```

Ruff (0.15.14)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)